### ISSUE TITLE
Global Setting Not responding

### ISSUE THREAD

**Issue by zein97** [pH7Software/pH7-Social-Dating-CMS#650]
Hi Pierre,

I'm not a programmer and want to use your software.

Below is what I already email to you in case you missed to read my email.

Hi Pierre,

I have just install your ph7cms 5.2.0

I go to modify the following in global setting but no effect at all after select:

1) Site Status: Maintenance (Offline)
2) Show "Powered By" link in footer: Disable (Not recommended)
3) Show the News Feed: Disable

Is it because it is unpaid version. If that the case, can u guide me on how to get the one time cost or any payment arrangement to subscribe so that I can customize the software (White label) for my website.

Please help and your advice is highly appreciated. Sorry because I'm totally not a programmer and just a laymen who interested to have my on dating website.

Regards
Zain 

Please advise me on the above matter as soon as possible sir because I really happy to try your software.

Thanks

**Comment by Undefined-Variables** [comment]
> I'm not a programmer and want to use your software.

This software is NOT suitable for anybody who starts off with this! Don't waste your time.

**Comment by zein97** [comment]
Ok. Thanks for for your feedback. Please close this issue

**Comment by Undefined-Variables** [comment]
@zein97 I was not being funny I was just being realistic. If you seriously want to run a website that has users, creates its own dynamic pages and has a database that will hold your users personal data you have to fully commit to it and be confident in what you are doing and what your website is doing and how it works. 

What would you do when something goes wrong? You wouldn't know where to start looking. If you are serious start by learning in this order HTML, CSS , PHP, JavaScript, MySQL, and once you have understanding of those you need to get good with Unix CMD line and then the software such as Apache and finally you will need to study security and vulnerabilities. 

Then you are ready to own a website!

**Comment by zein97** [comment]
Yes sir. I will do as what you advised. In fact I always try to learn further. Coding is new field to me as an accountant.

Thanks for your good advise.

**Comment by Undefined-Variables** [comment]
Books are great you can take them anywhere and Google knows absolutely everything :)

**Comment by zein97** [comment]
Yes sir. Absolutely correct. :)

**Comment by polynamaude** [comment]
@zein97 Maybe you shall confront your expectation with the real world of community based support and free software philosophy.

It's not by pushing people that you'll get anything done and at best you will get ignored.
@pH-7 Offer this software for free, without any commercial benefits. You say "Please answer me ! I need this ! I have big plans ! I want it now !"... Just here is the best reason to get ignored.

https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki
https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki/How-to-ask-for-help

**Comment by zein97** [comment]
Noted with thanks polynamaude.

Already give star in which I not aware of before. Thank for acknowledge me.

Keep up the good work to all of you here. I will try to explore this amazing software and learn the code. Hopefully I can contribute something in future.

**Comment by simon1967** [comment]
The guy's who have answered are fantastic. But I had problems, Check your host and the php setting it needs to be as per configuration also your server setting need to be as per configuration which can be found in your admin panel under documentation under requirements, Once this is set up re install a clean new copy , I am not a code person just learning, I am afraid my knowledge was in BASIC in 1981 so this is all new to me too. But this may just settle it down enough for you to learn more.
Hope I am not stepping on anyones toes here.
Simon

**Comment by simon1967** [comment]
Just to add the settings in this document it says php5.6 you need php 7.0 or above
Simon

**Comment by Undefined-Variables** [comment]
> I am afraid my knowledge was in BASIC in 1981 so this is all new to me too. But this may just settle it down enough for you to learn more.

@simon1967 I was -4 in 1981 so my knowledge was pritty basic too :)

**Comment by simon1967** [comment]
It was stone age Sir clive Sinclair ZX81 I was 14 yrs old i went from that to the spectrun, I was writing programs in good old BASIC. then Comadore 64, then a stupdity part jupiter ace this was FORTH then bbc model b 5/14 real floppy then Dragon 32 then my first desktop 88 mhz processor. I saved up and got my first 60gb hard drive and told myself this will last for ever i will never fill it haha. Then I got married and left it all alone :( but now I am ready to learn again haha or at least trying
Simon
